modules/operator-georepl-site-removal.adoc

```diff
@@ -0,0 +1,59 @@
+:_content-type: PROCEDURE
+[id="operator-georepl-site-removal"]
+= Removing a geo-replicated site from your {productname} Operator deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You are logged into {ocp}.
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
+
+.Procedure 
+
+. Enter the following command to identify your `Quay` database pods:
++
+[source,terminal]
+----
+$ oc get pod -l quay-component=postgres -n  <quay-namespace> -o jsonpath='{.items[0].metadata.name}'
+----
```

Comment:
it's not necessary to get the Quay Database POD, customers only need to rsh to the Quay App POD to execute the command to remove site